apps/nextjs-website/react-components/components/Header/Header.tsx

```diff
@@ -0,0 +1,124 @@
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import { useEffect, useState } from 'react';
+import { HamburgerMenu } from './components/HamburgerMenu';
+import { Navigation } from './components/Navigation';
+import { Content } from './components/Title';
+import { HeaderProps } from '@react-components/types/Header/Header.types';
+import { CtaButtons } from '../common/Common';
+import { BackgroundColor } from '../common/Common.helpers';
+
+const Header = ({
+  product,
+  theme,
+  menu,
+  ctaButtons,
+  avatar,
+  beta,
+  logo,
+}: HeaderProps) => {
+  const [headerOpen, setHeaderOpen] = useState(false);
+
+  const openHeader = () => {
+    setHeaderOpen(true);
+  };
+
+  const closeHeader = () => {
+    setHeaderOpen(false);
+  };
+
+  const onResize = () => {
+    closeHeader();
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', onResize);
```

Comment:
```suggestion
    window.addEventListener('resize', closeHeader);
```
Following above suggestion.